Source/Diagnostics/ComputeDiagFunctors/BackTransformFunctor.cpp

```diff
@@ -185,7 +206,12 @@ BackTransformFunctor::InitData ()
 
     for (int i = 0; i < m_varnames.size(); ++i)
     {
+#ifdef WARPX_DIM_RZ
+        const int field_id = i / (WarpX::n_rz_azimuthal_modes*2-1);
```

Comment:
Similarly to the comment above:
```suggestion
        const int field_id = i / WarpX::ncomps;
```